src/main/java/com/nylas/NeuralMessageOptions.java

```diff
@@ -3,6 +3,13 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Object representing configuration options for some Neural endpoints
+ * Currently only signature extraction and clean conversations support these configurations
+ *
+ * https://developer.nylas.com/docs/intelligence/signature-extraction/#parse-signature-request
+ * https://developer.nylas.com/docs/intelligence/clean-conversations/#clean-conversations-configuration
```

Comment:
Javadoc uses html (old) when viewing.  When I view the source code, this looks great.  My other use case is when I hover over a method, my IDE shows the HTML formatted version that puts it all on a single line, and doesn't make the links clickable.  I recommend using some basic html formatting to separate the paragraphs, and make the links clickable.  One example to see in the SDK is Files.download

Reference:
https://stackoverflow.com/questions/5260368/which-tag-should-be-used-as-paragraph-separator-in-javadoc